subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/plotting/error_report.py

```diff
@@ -95,8 +98,7 @@ def _get_test_setup(entry):
 
     sutest_ocp_version = entry.results.sutest_ocp_version
 
-    version_ts = entry.results.rhods_info.createdAt.strftime("%Y-%m-%d") \
-        if entry.results.rhods_info.createdAt else entry.results.rhods_info.createdAt_raw
+    version_ts = entry.results.start_time.strftime("%Y-%m-%d")
```

Comment:
no, the `version_ts` is the timestamp of RHODS builds, not the timestamp of the test :)